### PR TITLE
4752 Deleting cache

### LINKED ISSUE #4752: Outdated Bounds for LightweightTracker
Steps to reproduce:
1. Open `lightweight_test_tracks.rep`
2. Edit the properties of the first element of the Track "FisherOne_002" (like in the picture)
image
3. Change the Degrees of the location from 0 to 10
4. Apply all the changes.
5. Fit to Windows (It should show all the elements)

### PR DESCRIPTION
**Pull request by saulhidalgoaular**
Fix #4752

### CODE REVIEW

**Comment by IanMayo** [comment]
@saul - do you know there isn't an impact from removing the cached area in Plottables?

**Comment by saulhidalgoaular** [comment]
> @saul - do you know there isn't an impact from removing the cached area in Plottables?

Good afternoon. I have confirmed that it is needed to fix the bug. It is a cache that is not being validated after modifying the elements.

File: org.mwc.cmap.legacy/src/MWC/GUI/Plottables.java [hunk 1]
```diff
@@ -347,15 +347,8 @@ public synchronized void paint(final CanvasType dest)
...
-		{
-			// yup, get on with it...
-			_myArea = recalculateAreaCovered();
-		}
-
-		return _myArea;
-
+		// yup, get on with it...
+		return recalculateAreaCovered();
 	}
```
Comment (IanMayo) [inline org.mwc.cmap.legacy/src/MWC/GUI/Plottables.java]: so, @saulhidalgoaular  - shouldn't we remove the `_myArea` field?

Comment (saulhidalgoaular) [reply org.mwc.cmap.legacy/src/MWC/GUI/Plottables.java]: oh yes. 1 min

**Comment by IanMayo** [comment]
Hey @saulhidalgoaular  - the build isn't happy :-)

**Comment by IanMayo** [comment]
@helenayele  - we've merged a whole bunch of PRs today.  Some of them are quite old, and there's a chance something has changed in Debrief since they were originally tested.

So, while testing this PR (which relates to storing the bounds of items), it would be good to do some double-checking more widely across D-Lite and Full Debrief.  Just so we know we're on a good foundation before we do an automated code cleanup.

**Pull request merged**